src/libraries/System.Net.HttpListener/src/System/Net/Managed/HttpConnection.cs

```diff
@@ -100,7 +101,11 @@ public HttpConnection(Socket sock, HttpEndPointListener epl, bool secure, X509Ce
             }
 
             _timer = new Timer(OnTimeout, null, Timeout.Infinite, Timeout.Infinite);
+
+#pragma warning disable SYSLIB0014 // ServicePointManager is obsolete
             _sslStream?.AuthenticateAsServer(_cert, true, (SslProtocols)ServicePointManager.SecurityProtocol, false);
+#pragma warning restore SYSLIB0014
```

Comment:
The `SslStream` part of this logic is currently unreachable - tracked by #19752

https://github.com/dotnet/runtime/blob/d7ae8c61f0937df4dcffe88ba96eae65fa523a78/src/libraries/System.Net.HttpListener/src/System/Net/Managed/HttpEndPointListener.cs#L56-L60

https://github.com/dotnet/runtime/blob/d7ae8c61f0937df4dcffe88ba96eae65fa523a78/src/libraries/System.Net.HttpListener/src/System/Net/Managed/HttpListener.Certificates.cs#L20-L24

https://github.com/dotnet/runtime/blob/d7ae8c61f0937df4dcffe88ba96eae65fa523a78/src/libraries/System.Net.HttpListener/src/System/Net/Managed/HttpEndPointListener.cs#L115-L119